odoo/addons/base/models/ir_profile.py

```diff
@@ -108,3 +118,25 @@ def set_profiling(self, profile=None, collectors=None, params=None):
             'collectors': request.session.profile_collectors,
             'params': request.session.profile_params,
         }
+
+
+class EnableProfilingWizard(models.TransientModel):
+    _name = 'base.enable.profiling.wizard'
+    _description = "Enable profiling for some time"
+
+    duration = fields.Selection([
+        ('minutes_5', "5 Minutes"),
+        ('hours_1', "1 Hour"),
+        ('days_1', "1 Day"),
+        ('months_1', "1 Month")
```

Comment:
trailing comma

Reply:
shouldn't `duration` be required?

Reply:
Actually, yes and no, you can define the expiration by and, the duration is just a 'hack' to quiclky select a duration instead of a date. This part went actually through a few iterations and I may have lost my track, but it looked usable and I stopped there.

Reply:
Actually I tried the onchange solution to reset the duration to False but the problem is that the onchange will trigger the compute and reset the date (if i remember well)

I also tried to add an inverse (strore=True and inverse, lol) but it wasn't working either.

In this final version if you select a duration, it works fine, same if you change the expiration, It is juste weird if you select a duration then an expiration, the duration is inconsistent with the expiration but since the expiration is used it should be the expected behaviour in all cases.  

Actually it would maybe make more sence to have buttons to set the duration instead of a selection field, or to make the duration required and the expiration readonly, but I think it works fine like that. (It remains a debug feature)